settings.gradle.kts

```
@@ -75,5 +79,6 @@ project(":appyx-components:experimental:promoter:common").name = "promoter"
 project(":appyx-components:experimental:puzzle15:common").name = "puzzle15"
 project(":appyx-components:experimental:puzzle15:web").name = "puzzle15-web"
 project(":appyx-components:internal:test-drive:common").name = "test-drive"
+project(":demos:appyx-navigation:web").name = "navigation-web"
```

We must not touch platform-specific artifacts. Will be generated automatically. We just ned to rename common modules so the namings do not clash
```suggestion
project(":demos:appyx-navigation:common").name = "appyx-navigation"
```

---

note that this is the demos common module, not the appyx-navigation library - so probably needs to by ```project(":appyx-navigation:common").name = "appyx-navigation"``` for the navigation module.

For the demo module I was looking at what was done for the puzzle15 component, which has both common and web modules defined in there. Not sure what is required...

---

Since we're not publishing demos, I don't think we need to rename the module